src/routes/wallet/index.js

```diff
@@ -9,10 +9,17 @@ app.use('/*', apiKeyAuth);
 app.onError(errorHandler);
 
 app.get('/:phone_number', async (c) => {
-  const phoneNumber = c.req.param('phone_number');
+  const phoneNumber = c.req.param('phone_number').replace('+', '');
 
   const balance = await fetchWalletBalance(phoneNumber);
   return c.json({ balance });
 });
 
+app.get('/:phone_number/address', async (c) => {
+  const phoneNumber = c.req.param('phone_number').replace('+', '');
```

Comment:
same comment, this parsing should be done at the service level `fetchWalletAddress`